app/queries/articles/active_threads_query.rb

```diff
@@ -0,0 +1,32 @@
+module Articles
+  class ActiveThreadsQuery
+    DEFAULT_OPTIONS = {
+      tags: ["discuss"],
+      time_ago: nil,
+      count: 10
+    }.with_indifferent_access.freeze
+
+    MINIMUM_SCORE = -4
+
+    def self.call(relation: Article.published, options: {})
+      options = DEFAULT_OPTIONS.merge(options)
+      tags, time_ago, count = options.values_at(:tags, :time_ago, :count)
+
+      relation.limit(count)
+      relation = if time_ago == "latest"
+                   relation.order(published_at: :desc).where(score: MINIMUM_SCORE..).presence ||
+                     relation.order(published_at: :desc)
+                 elsif time_ago
+                   relation.order(comments_count: :desc)
+                     .where(published_at: time_ago.., score: MINIMUM_SCORE..).presence ||
+                     relation.order(comments_count: :desc)
+                 else
+                   relation.order("last_comment_at DESC NULLS LAST")
+                     .where(published_at: (tags.present? ? 5 : 2).days.ago.., score: MINIMUM_SCORE..).presence ||
```

Comment:
```
.where(published_at: (tags.present? ? 5 : 2).days.ago.., score: MINIMUM_SCORE..).presence ||
```
Does the distinction between whether an article was `published_at` 5 or 2 days depending on its tags matter here? I don't fully understand why this check is necessary and I'm thinking that we could do without it if it doesn't serve a greater purpose. Maybe rather than checking whether tags are present and then determining what `published_at` must be greater than, we could instead achieve this another way.

Reply:
Not sure either, I was able to trace it to [this old commit](https://github.com/forem/forem/commit/e052e63d340b8f04e70692082fb2ac7069ee06f8) but it predates the public repo. Maybe @benhalpern has additional context

Reply:
Hmm, thanks, @rhymes! ✨